src/math/Color.js

```diff
@@ -372,10 +372,16 @@ THREE.Color.prototype = {
 
 	},
 
-	toArray: function () {
+	toArray: function( array, offset ) {
 
-		return [ this.r, this.g, this.b ];
+		if ( array === undefined ) array = [];
+		if ( offset === undefined ) offset = 0;
 
+		array[ offset + 0 ] = this.r;
```

Comment:
Ok, here it is:

https://github.com/Wandalen/three.js/commit/f8579eb07893b5f9fdf5f4bb2610f5575dea215e